src/cli/ionic-config.ts

```diff
@@ -25,7 +30,10 @@ export async function readConfig(): Promise<TelemetryConfig> {
     };
 
     await writeConfig(config);
-  } else if (!!config && !config['tokens.telemetry'].match(UUID_REGEX)) {
```

Comment:
This is the line that was failing. If the config file existed, but 'tokens.telemetry' either:
- existed and wasn't a string
- did not exist at all

we would call `match()` on "something" that didn't have it in it's prototype chain

Reply:
It should be safe to do the inverse and call `RegExp.test()` with "something" as an argument, i.e. `!UUID_REGEX.test(config['tokens.telemetry'])`.

Reply:
Ahhh good point - done in https://github.com/ionic-team/stencil/pull/3014/commits/fc90181e3c2a600a9efcdd8a87aaf77bf70628ea